submatrix/homeserver.yaml.template

```diff
@@ -0,0 +1,37 @@
+server_name: "${MATRIX_HOSTNAME}"
+pid_file: /homeserver.pid
+listeners:
+  - port: 8008
+    tls: false
+    type: http
+    bind_addresses: ['::1', '127.0.0.1']
+    resources:
+      - names:
+        - client
+        compress: false
+  - path: /run/socks/submatrix.sock
+    type: http
+    x_forwarded: true
+    resources:
+      - names:
+        - client
+        - federation
+        compress: true
```

Comment:
perhaps compress should be off? I think that was maybe correct when it was doing its own tls (i.e. not behind a proxy), but now that it is behind nginx maybe it should be `compress: false`